nixos/modules/services/databases/influxdb.nix

```diff
@@ -68,7 +68,7 @@ let
 
     collectd = [{
       enabled = false;
-      typesdb = "${pkgs.collectd}/share/collectd/types.db";
+      typesdb = "${cfg.collectdPackage}/share/collectd/types.db";
```

Comment:
Is there a use-case for *not* using the data-only variant? If not, maybe just do that, at least for the initial version.